README.md

```diff
@@ -59,3 +59,25 @@ Currently, this Language Server only works for HTML, though its utility extends
 - If you want to debug the server as well use the launch configuration `Attach to Server`
 - In the [Extension Development Host] instance of VSCode, open a HTML file.
   - Type `<div data-controller="|">`, place your cursor where the `|` is, hit Ctrl+Space and you should see completions.
+
+## Install instructions: Neovim
+
+- Install the `stimulus-language-server` package globally
```

Comment:
Maybe we can just reference the install instructions in the README here: 
https://github.com/marcoroth/stimulus-lsp/blob/main/server/README.md

or we reference the README in the `nvim-lspconfig` README:
https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#stimulus_ls

Reply:
Maybe better immediately reference the README in `nvim-lspconfig`? All information is there.

Reply:
I also added a new install instruction section on hotwire.io: https://hotwire.io/ecosystem/tooling/stimulus-lsp/install/neovim, it might make sense to also have it over there.